src/main/java/io/xpring/xrpl/model/XrpAccountDelete.java

```diff
@@ -63,7 +63,7 @@ static XrpAccountDelete from(AccountDelete accountDelete, XrplNetwork xrplNetwor
     ClassicAddress classicAddress = ImmutableClassicAddress.builder()
         .address(destination)
         .tag(destinationTag)
-        .isTest(xrplNetwork == XrplNetwork.TEST)
+        .isTest(xrplNetwork == XrplNetwork.TEST || xrplNetwork == XrplNetwork.DEV)
```

Comment:
i encountered the `isTest` setter on `XrpCheckCreate` and this felt reasonable to me. lmk what you think

Reply:
word, i'll wait and see if we use it more